include/wx/generic/private/grid.h

```diff
@@ -793,6 +793,19 @@ class wxGridDirectionOperations
     // boundary, i.e. is the first/last row/column
     virtual bool IsAtBoundary(const wxGridCellCoords& coords) const = 0;
 
+    // Check if the component of this point in our direction is
+    // valid, i.e. not -1
+    bool IsValid(const wxGridCellCoords& coords) const
+    {
+        return m_oper.Select(coords) != -1;
+    }
+
+    // Make the coordinates with the other component value of -1.
+    wxGridCellCoords MakeLineCoords(const wxGridCellCoords& coords) const
```

Comment:
I think this name might be more clear:
```suggestion
    wxGridCellCoords MakeWholeLineCoords(const wxGridCellCoords& coords) const
```